pkgs/development/tools/pretty-simple/default.nix

```diff
@@ -0,0 +1,3 @@
+{ haskell, haskellPackages, stdenvNoCC }:
+
+haskell.lib.justStaticExecutables haskellPackages.pretty-simple
```

Comment:
Could you set a maintainer here?

You can do it like the following:

https://github.com/NixOS/nixpkgs/blob/d571ad2748fd039f8fa58a7ed198c35bfccd1ef2/pkgs/development/tools/purescript/spago/default.nix#L6-L9

------------------------

Also, you'll have to enable the `buildexe` flag for pretty-simple, since it is not enabled by default.

Here's how I do it in an overlay I have:

```nix
pretty-simple-cli =
  with pkgs.haskell.lib;
  addBuildDepend
    (appendConfigureFlag pkgs.haskellPackages.pretty-simple "-fbuildexe")
    pkgs.haskellPackages.optparse-applicative;
```

Reply:
wow, tnx, it didnt generate `/bin` dir before (dont know why I didnt mention that), but now it does, tnx a lot!, everything works now